runtime/convertValues.go

```diff
@@ -613,6 +623,63 @@ func exportDictionaryValue(
 	return dictionary.WithType(exportType), err
 }
 
+func exportCompositeValueAsInclusiveRange(
+	v interpreter.Value,
+	inclusiveRangeType *sema.InclusiveRangeType,
+	inter *interpreter.Interpreter,
+	locationRange interpreter.LocationRange,
+	seenReferences seenReferences,
+) (
+	cadence.InclusiveRange,
+	error,
+) {
+	compositeValue, ok := v.(*interpreter.CompositeValue)
+	if !ok {
+		// InclusiveRange is stored as a CompositeValue.
+		panic(errors.NewUnreachableError())
+	}
+
+	getNonComputedField := func(fieldName string) (cadence.Value, error) {
+		fieldValue := compositeValue.GetField(inter, locationRange, fieldName)
+		if fieldValue == nil {
+			// Bug if the field is absent.
+			panic(errors.NewUnreachableError())
+		}
+
+		return exportValueWithInterpreter(
+			fieldValue,
+			inter,
+			locationRange,
+			seenReferences,
+		)
+	}
+
+	startValue, err := getNonComputedField(sema.InclusiveRangeTypeStartFieldName)
+	if err != nil {
+		return cadence.InclusiveRange{}, err
+	}
```

Comment:
I am not sure how to create this scenario in the tests. We need the export of an integer value (`exportValueWithInterpreter` in above function) to fail to be able to test this failure path.